lib/Language/defaults.js

```diff
@@ -0,0 +1,21 @@
+const language = {
+  AddData: "Add Data",
+  EmptyWorkbenchMessage: `
+      <div>Your workbench is empty</div>
+      <p>
+        <strong>Click &apos;Add data&apos; above to:</strong>
+      </p>
+      <ul>
+        <li>Browse the Data Catalogue</li>
+        <li>Load your own data onto the map</li>
+      </ul>
+      <p>
+        <Icon glyph={Icon.GLYPHS.bulb} />
```

Comment:
This works???! 😮

Reply:
Answering my own question: probably not, based off the bulb missing in the screenshot 😆

Reply:
Answering even more, I just read the actual PR description. Nevermind me?

Reply:
You make me laugh @soyarsauce!